observation_portal/proposals/migrations/0008_auto_20210628_0304.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.23 on 2021-06-28 03:04
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('proposals', '0007_remove_timeallocation_instrument_type'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='proposal',
+            name='tags',
+            field=django.contrib.postgres.fields.ArrayField(base_field=models.CharField(max_length=255), blank=True, default=list, help_text='List of strings tagging this proposal', size=None),
+        ),
+        migrations.AlterField(
```

Comment:
This migration is here because it was a created from a previous update (not on this branch) that didn't have a migration along with it.

Reply:
strange, must be from me but I thought I created the migrations at the end before deploying. Oh well